fbw-common/src/systems/shared/src/GPUManagement.ts

```diff
@@ -28,10 +28,19 @@ export class GPUManagement implements Instrument {
 
   private readonly groundVelocity = ConsumerSubject.create(this.sub.on('msfs_ground_velocity'), 0);
 
+  private readonly cameraState = ConsumerSubject.create(this.sub.on('msfs_camera_state'), 0);
```

Comment:
Would be better to just read the simvar directly while only we’re waiting for the init, else we pay the cost for it every frame for the entire flight.